numpy/core/fromnumeric.py

```diff
@@ -2230,6 +2239,13 @@ def amax(a, axis=None, out=None, keepdims=np._NoValue):
         sub-class' method does not implement `keepdims` any
         exceptions will be raised.
 
+    initializer : scalar, optional
+        The initializer for this maximum. Can also be used to compute the
```

Comment:
See above. Maybe "Minimum value for the maximum. Needs to be present to allow calculation on an empty slice."